src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -0,0 +1,214 @@
+use crate::config::{DataType, SinkConfig, SinkContext, SinkDescription};
+use crate::event::{Metric, MetricValue};
+use crate::http::HttpClient;
+use crate::sinks::gcp;
+use crate::sinks::util::buffer::metrics::MetricsBuffer;
+use crate::sinks::util::http::{BatchedHttpSink, HttpSink};
+use crate::sinks::util::{BatchConfig, BatchSettings, TowerRequestConfig};
+use crate::sinks::{Healthcheck, VectorSink};
+use crate::tls::{TlsOptions, TlsSettings};
+use crate::Event;
+use chrono::{DateTime, Utc};
+use futures::sink::SinkExt;
+use futures::FutureExt;
+use http::header::AUTHORIZATION;
+use http::{HeaderValue, Uri};
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+
+#[derive(Deserialize, Serialize, Debug, Clone, Default)]
+#[serde(deny_unknown_fields)]
+pub struct StackdriverConfig {
+    pub project_id: String,
+    pub resource: gcp::GcpTypedResource,
+    pub credentials_path: Option<String>,
+    #[serde(default = "default_metric_namespace_value")]
+    pub default_metric_namespace: String,
```

Comment:
All the other components that have a default namespace for metrics use just `default_namespace` for the setting. Please rename this and the docs for conformity.

Reply:
Done